Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -147,6 +147,10 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		// default banner
 		m_Bitmap.LoadFile(StrToWxStr(File::GetThemeDir(SConfig::GetInstance().theme_name)) + "nobanner.png", wxBITMAP_TYPE_PNG);
 	}
+
+	if (isElfDol(_rFileName))
```

Comment:
This should be an `else if` branch on the `if (LoadFromCache())` if statement at line 74.

e.g.

```cpp
if (LoadFromCache())
{
    // Code here
}
else if (IsElfOrDol())
{
}
else
{
    // Wii/GC code here.
}
```